api/tools/src/main/java/org/pmiops/workbench/tools/ExportWorkspaceData.java

```diff
@@ -190,15 +142,79 @@ public CommandLineRunner run(
       final CustomMappingStrategy mappingStrategy = new CustomMappingStrategy();
       mappingStrategy.setType(WorkspaceExportRow.class);
 
-      StatefulBeanToCsv beanWriter =
-          new StatefulBeanToCsvBuilder(writer).withMappingStrategy(mappingStrategy).build();
+      try (FileWriter writer =
+          new FileWriter(opts.getOptionValue(exportFilenameOpt.getLongOpt()))) {
+        StatefulBeanToCsv beanWriter =
```

Comment:
nit: the convention I like for builders is to break after the `.builder()` or constructor call in this case, then after every argument, and finally have `build()` on one line.

In this case, you could even keep going and call `.write()` on the next line write after `.build()`, since I don't think there's that much to see in the debugger for this kind of class. But I'd probably do it the way you have it here.